torch/_torch_docs.py

```diff
@@ -8930,6 +8930,54 @@ def merge_dicts(*dicts):
 """.format(**factory_common_args))
 
 
+add_docstr(torch.kaiser_window, """
+kaiser_window(window_length, beta, periodic=True, *, dtype=None, \
+layout=torch.strided, device=None, requires_grad=False) -> Tensor
+""" + r"""
+Kaiser window function.
+
+.. math::
+    w[n] = I_0 \left( \beta \sqrt{1 - \frac{4n^2}{(M-1)^2}} \right) / I_0( \beta )
```

Comment:
Can we rewrite this in terms of the inputs, which don't involve `n` or `M`?

```
w_i = ...
```

and then ditch the inequality on `n` below and the definition of `M`? If we do want to define `M` for clarity let's define it BEFORE it's used in the expression.

Reply:
See the documentation for `torch.sub` for an example using of the mathematical expression using underscore notation:

https://pytorch.org/docs/master/generated/torch.sub.html?highlight=sub#torch.sub

Reply:
Speaking of defining things before they're used, I know we know what `i0` is (because you just made it), but this should define what `i0` and link to its documentation.